specification/metrics/semantic_conventions/http-metrics.md

```diff
@@ -54,6 +54,8 @@ operations. By adding HTTP attributes to metric events it allows for finely tune
 
 ### Metric: `http.server.duration`
 
+**Status**: [Experimental, Feature-freeze][DocumentStatus]
```

Comment:
It's unrelated to this PR, but I wonder if we shouldn't adapt our semconv tool to also generate this. We have the field for it in the [YAML schema ](https://github.com/open-telemetry/build-tools/blob/main/semantic-conventions/src/opentelemetry/semconv/model/semantic_attribute.py#L38). Except we don't have "feature-freeze" but I guess we could add that.

Reply:
@lmolkova proposed this in https://github.com/open-telemetry/build-tools/pull/170 but this is pending on https://github.com/open-telemetry/opentelemetry-specification/issues/3459 I think.

Reply:
@lmolkova proposed this in https://github.com/open-telemetry/build-tools/pull/170 but this is pending on https://github.com/open-telemetry/opentelemetry-specification/issues/3459 I think.